### PR TITLE
PUBDEV-6378 - Generic model release notes (PUBDEV-6136)

### PR DESCRIPTION
**Pull request by Pscheidl**
### Release notes PR
JIRA: [https://0xdata.atlassian.net/browse/PUBDEV-6378](https://0xdata.atlassian.net/browse/PUBDEV-6378)

### PR that is actually being documented
JIRA: [PUBDEV-6136](https://0xdata.atlassian.net/browse/PUBDEV-6136)
GitHub PullRequest: https://github.com/h2oai/h2o-3/pull/3274

### CODE REVIEW

File: h2o-docs/src/product/data-science/generic.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    data <- h2o.importFile(path = 'training_dataset.csv')
+    cols <- c("Some column", "Another column")
+    model.original <- h2o.glm(x=cols, y = "response", training_frame = data)    
+
+    path <- "/path/to/model/directory"
+    mojo.destination <- h2o.download_mojo(model = model.original, path = path)
+    
+    # Only import or upload MOJO model data, do not initialize the generic model yet
+    imported_mojo_key <- h2o.importFile(mojo.destination, parse = FALSE)
```
Comment (michalkurka) [inline h2o-docs/src/product/data-science/generic.rst]: Would be nice to combine the next 2 lines into a single convenient function. The users shouldn't need to worry about parse = FALSE

+ the mojo should be deleted from memory when the model gets deleted

File: h2o-docs/src/product/data-science/generic.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+   .. code-block:: python
+
+    data = h2o.import_file(path='training_dataset.csv')
+    glm = H2OGeneralizedLinearEstimator()
+    glm.train(x = ["Some column", "Another column"], y = "response", training_frame=data)
+
+    path = '/path/to/model/directory/model.zip'
+    drf.download_mojo(path)
+    
+    imported_mojo_key = h2o.lazy_import(file)
```
Comment (michalkurka) [inline h2o-docs/src/product/data-science/generic.rst]: same as above - make it easy for the users

File: h2o-docs/src/product/data-science/generic.rst (hunk 1 above)
Comment (Pscheidl) [reply h2o-docs/src/product/data-science/generic.rst]: There is such convenience function. This is only the advanced example when the MOJO already is in-memory for ANY reason. Look above.

File: h2o-docs/src/product/data-science/generic.rst (hunk 2 above)
Comment (Pscheidl) [reply h2o-docs/src/product/data-science/generic.rst]: Same as above, read the previous part of the documentation. The convenience function comes first, then this.

File: h2o-docs/src/product/data-science/generic.rst (hunk 1 above)
Comment (Pscheidl) [reply h2o-docs/src/product/data-science/generic.rst]: See the `Importing a Generic Model` section.

File: h2o-docs/src/product/data-science/generic.rst (hunk 2 above)
Comment (Pscheidl) [reply h2o-docs/src/product/data-science/generic.rst]: See the `Importing a Generic Model` section

File: h2o-docs/src/product/data-science/generic.rst [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+.. example-code::
+   .. code-block:: r
+
+    data <- h2o.importFile(path = 'training_dataset.csv')
+    cols <- c("Some column", "Another column")
+    model.original <- h2o.glm(x=cols, y = "response", training_frame = data)    
+
+    path <- "/path/to/model/directory"
+    mojo.destination <- h2o.download_mojo(model = model.original, path = path)
+    model.generic <- h2o.genericModel(mojo.destination)
```
Comment (ledell) [inline h2o-docs/src/product/data-science/generic.rst]: My only comment would be to not use dots in object names (this goes against the [tidyverse style guide](https://style.tidyverse.org/), so I'd change this to `model_original`, `mojo_destination` and `model_generic`.

Comment (ledell) [reply h2o-docs/src/product/data-science/generic.rst]: Also ideally we would use the same object names in the R and Python examples (e.g. `glm` and `model` in Python vs `model_original` and `model_generic` in R) when we do side-by-side examples in the docs.

Comment (Pscheidl) [reply h2o-docs/src/product/data-science/generic.rst]: Resolved. Dots removed, variable names aligned.

**Comment by Pscheidl** [comment]
All comments addressed.

File: h2o-docs/src/product/data-science/generic.rst (hunk 2 above)
Comment (michalkurka) [reply h2o-docs/src/product/data-science/generic.rst]: ok, thanks for explaining

**Pull request merged**